src/Service/Resolvers/MsSqlQueryExecutor.cs

```diff
@@ -194,5 +197,59 @@ public override string GetSessionParamsQuery(HttpContext? httpContext, IDictiona
 
             return sessionMapQuery.ToString();
         }
+
+        /// <inheritdoc/>
+        public override async Task<DbResultSet> GetMultipleResultSetsIfAnyAsync(
```

Comment:
>GetMultipleResultSetsIfAnyAsync [](http://example.com/codeflow?start=48&length=31)

This implementation is necessary because we only support this functionality for mssql, correct?

Reply:
yes correct.